brave/brave-core
# Import extension settings

Resolves https://github.com/brave/brave-browser/issues/8847


- Import extension settings 
- Added tests to extensions importer

https://user-images.githubusercontent.com/2965009/188576413-a71aa528-0693-463e-b1a1-44b95110eaa3.mp4

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Install Metamask to Google Chrome
- Login/create wallet in MM
- Import extensions from Chrome to Brave
- Check that MM has same state in Brave as it was befor in Chrome
